src/coreclr/inc/corinfo.h

```diff
@@ -2082,9 +2081,16 @@ class ICorStaticInfo
             CORINFO_CLASS_HANDLE        classHnd
             ) { return false; }
 
-    // return the unmanaged calling convention for a PInvoke
-    virtual CorInfoUnmanagedCallConv getUnmanagedCallConv(
-            CORINFO_METHOD_HANDLE       method
+    // return the entry point calling convention for any of the following
+    // - a P/Invoke
+    // - a method marked with UnmanagedCallersOnly
+    // - a function pointer with the CORINFO_CALLCONV_UNMANAGED calling convention.
+    // May return CorInfoCallConvExtension::Managed when the method or call site doesn't
```

Comment:
There is nothing useful that the JIT can do with managed calling convention for unmanaged entrypoint.

I think this should be an error that is handled on the EE side by either silently translating the calling convention to platform default (I believe that it is what happens today in some cases) or by throwing exception.